Libraries/Text/TextInput/RCTBaseTextInputView.m

```diff
@@ -145,12 +145,20 @@ - (void)setAttributedText:(NSAttributedString *)attributedText
 
   textNeedsUpdate = ([self textOf:attributedTextCopy equals:backedTextInputViewTextCopy] == NO);
 
+  NSString *currentScreenreaderError = self.backedTextInputView.currentScreenreaderError;
+  NSString *previousScreenreaderError = self.backedTextInputView.previousScreenreaderError;
+  BOOL errorMessageRemoved = currentScreenreaderError == nil && ![currentScreenreaderError isEqualToString: previousScreenreaderError];
+  if (currentScreenreaderError == nil && errorMessageRemoved) {
+    BOOL validString = attributedText && [attributedText.string length] != 0;
+    NSString *lastChar = validString ? [attributedText.string substringFromIndex:[attributedText.string length] - 1] : @"";
+    self.backedTextInputView.accessibilityValue = attributedText.string;
```

Comment:
setting accessibilityValue to nil is enought to fix this regression. This logic has to be removed

https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619583-accessibilityvalue?language=objc

Reply:
https://github.com/fabriziobertoglio1987/react-native/commit/16d7d4d8494a97f162ad8429043f6b5f334fd019